test/inductor/test_torchinductor.py

```diff
@@ -5114,6 +5114,18 @@ def fn(a, b):
                 fn(a, b)
             assert "kernel_cpp_0" in (e.name for e in prof.profiler.function_events)
 
+        def test_input_is_view(self):
```

Comment:
To be more accurate, it only happens with an in-place view op and should be fine non-place view, right? Naming it as `test_input_is_inplace_view` is more appropriate?

Reply:
Right. This function name is updated.